src/picterra/client.py

```diff
@@ -287,13 +287,11 @@ def run_detector(self, detector_id, raster_id):
             }
         )
         assert resp.status_code == 201, resp.status_code
-        data = resp.json()
-        result_id = data['result_id']
-
-        self._wait_until_operation_completes(resp.json())
-        return result_id
+        operation_response = resp.json()
+        self._wait_until_operation_completes(operation_response)
+        return operation_response['operation_id']
```

Comment:
I believe here we should rather get the operation and return its results instead. I'd prefer not to have users store `operation_ids`, since they're meant to be volatile. That way, the `download_result_to_file` function below collapses to a simple `requests.get(result_url)`.